src/Init.ts

```diff
@@ -0,0 +1,88 @@
+import { execSync } from 'child_process';
+import { Errors } from './Types';
+
+import colors from 'colors'
+import fs from 'fs';
+import path from 'path';
+import yargs from 'yargs';
+
+
+export function checkRequirements() {
+    try {
+        return execSync('ffmpeg -version',
+        { stdio: ['pipe','pipe','ignore'] }).toString().split('\n')[0];
+    } catch (e) {
+        return null;
+    }
+}
+
+// TODO: implement check for absolute path for correct console logging
+export function createVideoDir(directory: string) {
+    if (!fs.existsSync(directory)) {
+        console.log('Creating output directory: ' +
+            process.cwd() + path.sep + directory);
+        fs.mkdirSync(directory);
+    }
+}
+
+
+export function handleSetup() {
+    process.on('unhandledRejection', (reason) => {
+        console.error(colors.red('Unhandled error!\nTimeout or fatal error, please check your downloads and try again if necessary.\n'));
+        console.error(colors.red(reason as string));
+    });
+
+    process.on('exit', (code) => {
+        if (code == 0) {
+            return
+        };
+        if (code in Errors)
+            console.error(colors.bgRed(`\nError: ${Errors[code]}\n`))
+        else
+            console.error(colors.bgRed(`\nUnknown exit code ${code}\n`))
+    });
+}
+
+
+export function parseArgs() {
```

Comment:
i'm not sure we can make a function for yargs, we are not supposed to call this more than once and someone could call this function in more places by accident